ci/wheel_smoke_test_cugraph_kg.py

```diff
@@ -13,7 +13,7 @@
 
 import cudf
 import cugraph
-import cugraph-kg
+import cugraph_pg
```

Comment:
I wasn't sure what this should be, as I see references to both `cugraph_pg`

https://github.com/rapidsai/cugraph-pg/blob/d0d101a53c89984841199c3555bde6ee5d8658be/python/cugraph-pg/pyproject.toml#L16

and `cugraph_kg`

https://github.com/rapidsai/cugraph-pg/blob/d0d101a53c89984841199c3555bde6ee5d8658be/conda/recipes/cugraph/meta.yaml#L11

Reply:
I don't even seen this import being used in the smoke test. I would ignore this for now if you can. Admin-merge or something like that. This package isn't even named in a legal way (it uses dashes in places that aren't allowed) so I would put this back on library devs to resolve first.

Reply:
Alright sure, I just reverted the changes to this file. It won't require an admin-merge, as CI isn't set up here yet.